src/ctia/entity/feed/schemas.clj

```diff
@@ -81,7 +82,7 @@
     owner :- s/Str
     groups :- [s/Str]
     prev-object :- (s/maybe StoredFeed)]
-   (let [long-id (short-id->long-id id)
+   (let [long-id (short-id->long-id id p/get-in-global-properties)
```

Comment:
This function body will be wrapped in a function taking services. For now, using global properties.